code/modules/hallucinations/effects/chaser_hallucinations.dm

```diff
@@ -0,0 +1,114 @@
+/**
```

Comment:
I appreciate moving this out to a new file for organization sake, but I think it'll probably just end up causing conflicts for others. I appreciate the fix, but do you think you could do it in a later PR?